src/pages/pollingPage.vue

```diff
@@ -6,21 +6,36 @@
     </button>
```

Comment:
what is pollingPage? is this for single Poll or is this For Poll list page? Can you choose a better name @ShivanshSinghnnovative